docs/usage.md

```diff
@@ -124,3 +124,29 @@ add_action( 'eg_midnight_log', 'eg_log_action_data' );
 ```
 
 Note that the `as_has_scheduled_action()` function was added in 3.3.0: if you are using an earlier version, you should use `as_next_scheduled_action()` instead. For more details on all available API functions, and the data they accept, refer to the [API Reference](/api/).
+
+## Passing arguments
```

Comment:
Should this be a level 3 header? Seems like it's still related to "Scheduling an action" above...

Reply:
🤔 I initially felt it was different—but I see your point.

Reply:
Updated in https://github.com/woocommerce/action-scheduler/pull/969/commits/c4420712d82a3d6f0a41c1ad15bf9bf47e99c422.